### PR TITLE
Models and relationship creation

### PR DESCRIPTION
**Pull request by inahsanjuan**

### CODE REVIEW

File: app/User.php [hunk 1]
```diff
@@ -3,24 +3,37 @@
...
-    /**
-     * The attributes that should be hidden for arrays.
-     *
-     * @var array
-     */
-    protected $hidden = [
-        'password', 'remember_token',
-    ];
-}
+    protected $fillable = [ 'name', 'email', 'password',];
```
Comment (nguyenthanhtung88) [inline app/User.php]: 1. Remove space after `[`
2. If you choose convention to add `,` at the end item of array, you should make change for all array defined above, or for simple just remove `,` before `]`. Please make decision to your own convention.

File: app/User.php [hunk 2]
```diff
@@ -3,24 +3,37 @@
...
+    protected $hidden = [ 'password', 'remember_token', ];
```
Comment (nguyenthanhtung88) [inline app/User.php]: https://github.com/framgia/fels_168/pull/4/files#r72927661

File: app/User.php [hunk 3]
```diff
@@ -3,24 +3,37 @@
...
+
+    public function roles()
+    {
+        return $this->belongsToMany(Role::class);
+    }
+
+    public function followers()
+    {
+        return $this->belongsToMany(User::class,'following_id', 'follower_id');
```
Comment (nguyenthanhtung88) [inline app/User.php]: Convention error: we need one space after `,`

File: app/WordAnswer.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use App\Word;
+
+class WordAnswer extends Model
+{
+	protected $fillable = ['content', 'is_correct', 'word_id'];
```
Comment (nguyenthanhtung88) [inline app/WordAnswer.php]: Wrong indent, it should back 1 indent

File: app/User.php (hunk 1 above)
Comment (bkintanar) [inline app/User.php]: I believe `,` can be added if you have multiple lines of array. but for a one-line array, last `,` is not needed.

**Comment by nguyenthanhtung88** [comment]
OK

**Pull request merged**